server/utils/nunjucksSetup.ts

```diff
@@ -84,6 +84,29 @@ export function registerNunjucks(app?: express.Express): Environment {
     return dateToFormat ? format(parseISO(dateToFormat), dateFormat) : null
   })
 
+  njkEnv.addFilter('displayAge', (dateOfBirth: string) => {
+    let ageString
+
+    const today = new Date()
+    const dob = new Date(dateOfBirth)
+    const years = today.getFullYear() - dob.getFullYear()
+    if (years > 1) {
+      ageString = `${years} years old`
+    } else if (years === 1) {
+      ageString = '1 year old'
+    } else if (years < 1) {
+      const months = today.getMonth() - dob.getMonth()
+
+      if (months > 1) {
```

Comment:
Perhaps very unlikely to occur, but if a baby less than one month old then I guess this would show `0 month old` but should it be `0 months`?

Reply:
Just re-checked and realised that was mentioned on the ticket, will go back and add it now, Thanks!

Reply:
That looks like it should do the trick. For readability, could consider flipping the condition so instead of:
```if (months > 1 || months === 0) ...```
 it's
```if (months === 1) { // use singular } else { // use plural }``` 

Could also cover these extra cases in `nunjucksSetup.test.ts`